src/main/java/com/porpapps/wealthwatchapi/model/Portfolio.java

```diff
@@ -0,0 +1,16 @@
+package com.porpapps.wealthwatchapi.model;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+public class Portfolio {
+    private Integer id;
+    private User user;
+    private String name;
+    private Enum currency;
+    private LocalDate created;
+    private LocalDate updated;
```

Comment:
We also want to save time and this field is DATETIME in the DB script, so this should be LocalDateTime instead.